pyscitt/pyscitt/cli/retrieve_signed_claims.py

```diff
@@ -16,6 +16,7 @@ def retrieve_signed_claimsets(
     from_seqno: Optional[int],
     to_seqno: Optional[int],
     service_trust_store_path: Optional[Path],
+    dont_embed_receipt: bool = False,
```

Comment:
I explicitly used a double negative here to minimize breaking changes. By default, the current CLI retrieves all claims with embedded receipts. Introducing an optional flag for embedding receipts would mean that clients would start getting claims without embedded receipts by default, unless they use the new flag.

While considering this change, I preferred using a double negative to avoid breaking changes; however, if we are not concerned about it and prefer to make the CLI arguments clearer, I can definitely make the change to avoid the double negative.

Reply:
I would not be concerned with such a breaking change because getting the signature back without a receipt does not make much sense. I would consider it a feature because prior to that you needed to explicitly set the argument to get the embedded sig.

Reply:
Fair enough. Addressed in https://github.com/microsoft/scitt-ccf-ledger/pull/196/commits/d22216a3994850124c91db08b959fd06bb8aae73